deployment/terraform/strings.go

```diff
@@ -194,13 +193,13 @@ proxy_cache_use_stale timeout;
 proxy_cache_lock on;
 `
 
-const nginxSiteConfig = `
+const nginxSiteConfigTmpl = `
 upstream backend {
-%s
+{{.backends}}
   keepalive 256;
 }
 
-proxy_cache_path /var/cache/nginx levels=1:2 keys_zone=mattermost_cache:10m max_size=3g inactive=120m use_temp_path=off;
+proxy_cache_path /var/cache/nginx levels=1:2 keys_zone=mattermost_cache:{{.cacheObjects}} max_size={{.cacheSize}} inactive=60m use_temp_path=off;
```

Comment:
@agarciamontoro An additional note to include `nginx` recommendations as part of the reference architecture.

Reply:
I think it should be nginx + postgres config optimizations as well.